worker.go

```diff
@@ -22,7 +23,7 @@ func startPolling(awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter, poll
 func doWork(awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter) error {
 	defer func() error {
 		if r := recover(); r != nil {
-			log.Println("shit has hit the fan:", r)
+			log.Println("shit has hit the fan:", errors.Wrap(r.(error), "panic caused by"))
```

Comment:
Does this actually show the stacktrace where the error occurred? I would assume it's just the stack of this local recover function, but maybe I'm wrong?